src/dialect/mssql/mssql-driver.ts

```diff
@@ -132,6 +133,21 @@ class MssqlConnection implements DatabaseConnection {
     )
   }
 
+  async connect(): Promise<this> {
+    await new Promise((resolve, reject) => {
+      this.#connection.connect((error) => {
+        if (error) {
+          console.error(error)
```

Comment:
are we cool with logging these? normally underlying drivers do..

Reply:
Yep